### PR TITLE
Fix doc-nits and md-nits

### PR DESCRIPTION
**Pull request by tmshort**
Update documentation and README.md.

##### Checklist

- [X] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by nibanks — APPROVED** [review]

**Comment by tmshort** [comment]
Waiting until I can see that the "nits" CI is working. :)

**Comment by richsalz** [comment]
approved once the CI nits pass.  if not, then i guess we'll need another PR first :)

**Review by richsalz — APPROVED** [review]

**Comment by tmshort** [comment]
"check update" is expected to fail, as upstream forgot to do it before the beta.

**Comment by tmshort** [comment]
Grr... it's complaining about MD003... which I don't have issues with?!?

**Comment by tmshort** [comment]
I can't install the ruby gem version of mdl (my ruby is too old). But I reverted the headers...

**Comment by tmshort** [comment]
OK check-docs is passing... are we good @richsalz ?

**Comment by richsalz** [comment]
> OK check-docs is passing... are we good @richsalz ?

moral questions aside: yes :)

**Comment by tmshort** [comment]
Those are the two checks we'd expect to fail.

File: README.md [hunk 1]
```diff
@@ -38,53 +39,62 @@ On to the questions and answers.
...
-is from the original OpenSSL and what is for QUIC.
+  is from the original OpenSSL and what is for QUIC.
 - New versions are quickly created once upstream creates a new tag.
 - The use of git commands (such as "cherry") can be used to ensure that all changes
-have moved forward with minimal or no changes. You will be able to see "QUIC: Add X"
-on all branches and the commit itself will be nearly identical on all branches, and
-any changes to that can be easily identified.
+
+have moved forward with minimal or no changes. You will be able to see
+QUIC: Add X" on all branches and the commit itself will be nearly identical on all
```
Comment (kaduk) [inline README.md:61]: We seem to have unbalanced quotes here, now.

Comment (tmshort) [reply README.md:61]: Whoops

**Pull request merged**